theano/gpuarray/tests/test_basic_ops.py

```diff
@@ -396,7 +396,7 @@ def check(dtype, N, M_=None):
         k_symb = numpy.asarray(0)
         out = T.eye(N_symb, M_symb, k_symb, dtype=dtype)
         f = theano.function([N_symb, M_symb],
-                            out,
+                            T.stack(out),
```

Comment:
I guess it is correct.
I'm still not sure why stack gets transferred to GPU, since its output (the output of the function) is on CPU, though.